packages/client/src/app/game/views/game-map/game-map.component.ts

```diff
@@ -24,7 +24,7 @@ export class GameMapComponent implements OnInit, OnDestroy {
     Cesium.Cartesian3.fromDegrees(-73.985187, 40.758857, 1000);
   public static readonly DEFAULT_PITCH = -5;
 
-  @Input() private playersPositions: Observable<AcNotification>;
+  @Input() playersPositions: Observable<AcNotification>;
```

Comment:
ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-map/game-map.component.html (5,3): Property 'playe  Positions' is private and only accessible within class 'GameMapComponent'.